.cirrus.yml

```diff
@@ -539,7 +539,7 @@ rootless_integration_test_task:
     skip: *branches_and_tags
     depends_on:
         - unit_test
-    matrix: *fedora_vm_axis
+    matrix: *platform_axis
```

Comment:
I would respectfully ask this specific change be dropped from this PR and left up to #9905